autogen/agentchat/conversable_agent.py

```diff
@@ -640,7 +641,7 @@ def generate_oai_reply(
         # TODO: line 301, line 271 is converting messages to dict. Can be removed after ChatCompletionMessage_to_dict is merged.
         extracted_response = client.extract_text_or_completion_object(response)[0]
         if not isinstance(extracted_response, str):
-            extracted_response = extracted_response.model_dump(mode="dict")
+            extracted_response = model_dump(extracted_response)
```

Comment:
Does this work with Pydantic V1 also?

Reply:
Yes, this is from _pydantic.py, which checks for different pydantic versions and define the function